### PR TITLE
Feat/forms

### PR DESCRIPTION
**Pull request by MarouanLaroui**

### CODE REVIEW

File: src/components/forms/application/application-form.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  return (
+    <Formik
+      initialValues={{
+        mail: '',
+        password: '',
+      }}
+      validationSchema={applicationSchema}
+      onSubmit={async (data, { setSubmitting }) => {
+        setSubmitting(true);
```
Comment (NathanDjianMartin) [inline src/components/forms/application/application-form.tsx:40]: Put submit logic in external function

File: src/components/forms/company/company-form.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              variant="contained"
+              type="submit"
+            >
+              Sign up
+            </Button>
+
+            <Stack direction="row" justifyContent="center" width="100%" paddingTop="2rem">
+
+              <div>
+                {JSON.stringify(formik)}
```
Comment (NathanDjianMartin) [inline src/components/forms/company/company-form.tsx]: Remove that div

File: src/components/forms/worker/worker-form.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+            <Stack direction="row" justifyContent="center" width="100%" paddingTop="2rem">
+              <Button
+                disabled={formik.isSubmitting || !formik.isValid}
+                variant="contained"
+                type="submit"
+              >
+                Se connecter
+              </Button>
+              <div>
+                {JSON.stringify(formik)}
```
Comment (NathanDjianMartin) [inline src/components/forms/worker/worker-form.tsx]: Remove that

**Review by NathanDjianMartin — CHANGES_REQUESTED** [review]

File: src/components/forms/application/application-form.tsx (hunk 1 above)
Comment (MarouanLaroui) [reply src/components/forms/application/application-form.tsx:40]: Modified as suggested

File: src/components/forms/company/company-form.tsx (hunk 2 above)
Comment (MarouanLaroui) [reply src/components/forms/company/company-form.tsx]: Modified as suggested

File: src/components/forms/worker/worker-form.tsx (hunk 3 above)
Comment (MarouanLaroui) [reply src/components/forms/worker/worker-form.tsx]: Modified as suggested

File: src/components/forms/job-offer/job-offer-form.tsx [hunk 4]
```diff
@@ -122,6 +121,9 @@ export default function JobOfferForm() {
             >
               Se connecter
             </Button>
+            <div>
+              {JSON.stringify(formik)}
```
Comment (NathanDjianMartin) [inline src/components/forms/job-offer/job-offer-form.tsx]: REMOVE THAT

**Review by NathanDjianMartin — CHANGES_REQUESTED** [review]

**Review by NathanDjianMartin — APPROVED** [review]
LGTM

**Pull request merged**